src/components/header/search.jsx

```diff
@@ -69,10 +70,6 @@ export default function Search() {
 	const root = useRef(null);
 	const rendered = useRef(false);
 	const interactedWith = useRef(false);
-	const loadingBar =
-		typeof window !== 'undefined'
-			? document.querySelector('loading-bar')
-			: null;
```

Comment:
Shouldn't be problematic once deployed, due to prerendering, but annoying in dev. The `<loading-bar>` component renders in the page, so after the header, and won't exist by the time it's captured here.

Very superficial, but silly left over that should've been caught.